ee/dtr/user/manage-images/sign-images/index.md

```diff
@@ -7,53 +7,52 @@ redirect_from:
 - /ee/dtr/user/manage-images/sign-images/manage-trusted-repositories/
 ---
 
-2 Key components of the Docker Trusted Registry are the Notary Server and Notary
-Signer. These 2 containers give us the required components to use Docker Content
-Trust right out of the box. [Docker Content
-Trust](/engine/security/trust/content_trust/) allows us to sign image tags,
+Two key components of the Docker Trusted Registry are the Notary Server and the Notary
+Signer. These two containers offer the required components for using Docker Content
+Trust (DCT) out of the box. [Docker Content
+Trust](/engine/security/trust/content_trust/) allows you to sign image tags,
 therefore whoever pulls the image can validate that they are getting the image
-you create, or a forged one. 
+you created. 
```

Comment:
@ollypom I'm sorry my edits are delayed on this one, but I couldn't quite make sense of "a forged one" in this sentence. Let me know and I can rephrase.

Reply:
Hmm, I was referred v poorly to a untrusted / manipulated image.